### ISSUE TITLE
Error while generating response data

### ISSUE THREAD

**Issue by lkellar** [puffyboa/virtual-assistant#22]
I typed `v pause`, it began to generate response data, then I got this error.
``` 
✘ lucas@LucasPC  ~  v pause
Generating response data...
Traceback (most recent call last):
  File "/home/lucas/PycharmProjects/virtual-assistant/main.py", line 90, in <module>
    from responses import RESPONSES,offlineMode
  File "/home/lucas/PycharmProjects/virtual-assistant/responses.py", line 372, in <module>
    {"input": [".*temperature"]+list(itertools.chain.from_iterable([".+%s .*today" % s,".+%s out" % s] for s in syn('hot')))+list(itertools.chain.from_iterable([".+%s .*today" % s,".+%s out" % s] for s in syn('cold'))),
  File "/home/lucas/PycharmProjects/virtual-assistant/responses.py", line 33, in syn
    raise Exception("No synonyms grabbed for {}".format(word))
Exception: No synonyms grabbed for hot
Error in sys.excepthook:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/apport_python_hook.py", line 63, in apport_excepthook
    from apport.fileutils import likely_packaged, get_recent_crashes
  File "/usr/lib/python3/dist-packages/apport/__init__.py", line 5, in <module>
    from apport.report import Report
  File "/usr/lib/python3/dist-packages/apport/report.py", line 30, in <module>
    import apport.fileutils
  File "/usr/lib/python3/dist-packages/apport/fileutils.py", line 23, in <module>
    from apport.packaging_impl import impl as packaging
  File "/usr/lib/python3/dist-packages/apport/packaging_impl.py", line 24, in <module>
    import apt
  File "/usr/lib/python3/dist-packages/apt/__init__.py", line 23, in <module>
    import apt_pkg
ModuleNotFoundError: No module named 'apt_pkg'

Original exception was:
Traceback (most recent call last):
  File "/home/lucas/PycharmProjects/virtual-assistant/main.py", line 90, in <module>
    from responses import RESPONSES,offlineMode
  File "/home/lucas/PycharmProjects/virtual-assistant/responses.py", line 372, in <module>
    {"input": [".*temperature"]+list(itertools.chain.from_iterable([".+%s .*today" % s,".+%s out" % s] for s in syn('hot')))+list(itertools.chain.from_iterable([".+%s .*today" % s,".+%s out" % s] for s in syn('cold'))),
  File "/home/lucas/PycharmProjects/virtual-assistant/responses.py", line 33, in syn
    raise Exception("No synonyms grabbed for {}".format(word))
Exception: No synonyms grabbed for hot
```

**Comment by lkellar** [comment]
ok, this bug is thrown for every command I try to run

**Comment by lkellar** [comment]
Fixed with #23
